.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
remove this file

Reply:
done in https://github.com/magento/devdocs/pull/4827